examples/python/echo/README.md

```diff
@@ -35,20 +39,81 @@ Step 5 - Tail the logs of the veba-echo function
+2020/10/05 19:40:49 POST / - 200 OK - ContentLength: 0
 ```
 
-For readability, you can copy the JSON:
+For readability, you can copy the JSON data (everything in `{}` after `b'`) and format that using a JSON Linter website such as
+[https://jsonlint.com/](https://jsonlint.com/) or a program like `jq`:
 
 ```
-{"id":"6be1aa78-4e34-4697-87bd-fd189934804d","source":"https://vcenter.sddc-a-b-c-d.vmwarevmc.com/sdk","specversion":"1.0","type":"com.vmware.event.router/event","subject":"VmPoweredOffEvent","time":"2020-02-23T22:29:28.911840208Z","data":{"Key":303794,"ChainId":303792,"CreatedTime":"2020-02-23T22:29:28.226884Z","UserName":"VMC.LOCAL\\cloudadmin","Datacenter":{"Name":"SDDC-Datacenter","Datacenter":{"Type":"Datacenter","Value":"datacenter-3"}},"ComputeResource":{"Name":"Cluster-1","ComputeResource":{"Type":"ClusterComputeResource","Value":"domain-c8"}},"Host":{"Name":"10.20.32.4","Host":{"Type":"HostSystem","Value":"host-11"}},"Vm":{"Name":"Test","Vm":{"Type":"VirtualMachine","Value":"vm-1081"}},"Ds":null,"Net":null,"Dvs":null,"FullFormattedMessage":"Test on  10.20.32.4 in SDDC-Datacenter is powered off","ChangeTag":"","Template":false},"datacontenttype":"application/json"}
+$ pbpaste| jq .
+{
+  "data": {
+    "Key": 18968,
+    "ChainId": 18963,
+    "CreatedTime": "2020-10-05T19:40:49.134Z",
+    "UserName": "VSPHERE.LOCAL\\\\Administrator",
+    "Datacenter": {
+      "Name": "vcqaDC",
+      "Datacenter": {
+        "Type": "Datacenter",
+        "Value": "datacenter-2"
+      }
+    },
+    "ComputeResource": {
+      "Name": "cls",
+      "ComputeResource": {
+        "Type": "ClusterComputeResource",
+        "Value": "domain-c7"
+      }
+    },
+    "Host": {
+      "Name": "10.0.0.119",
+      "Host": {
+        "Type": "HostSystem",
+        "Value": "host-21"
+      }
+    },
+    "Vm": {
+      "Name": "test-01",
+      "Vm": {
+        "Type": "VirtualMachine",
+        "Value": "vm-57"
+      }
+    },
+    "Ds": null,
+    "Net": null,
+    "Dvs": null,
+    "FullFormattedMessage": "DRS powered On test-01 on 10.0.0.119 in vcqaDC",
+    "ChangeTag": "",
+    "Template": false
+  },
+  "datacontenttype": "application/json",
+  "id": "1f2a18c6-c788-4a9a-a2fb-8be0ec34330f",
+  "source": "https://10.0.0.28/sdk",
+  "specversion": "1.0",
+  "subject": "DrsVmPoweredOnEvent",
+  "time": "2020-10-05T19:40:49.508328Z",
+  "type": "com.vmware.event.router/event"
+}
 ```
 
-and format that using a JSON Linter website such as [https://jsonlint.com/](https://jsonlint.com/)
\ No newline at end of file
+## Build from scratch
+
+```bash
+
```

Comment:
Did you mean to have something here in the `Build from scratch` section?

Reply:
Fixed